environments/gym/web/dist/GymCartPole.js

```diff
@@ -0,0 +1,2 @@
+import{useCallback as m,useState as A}from"react";import{useDocumentKeypressListener as N,usePressedKeys as R,PlayObserver as S,useRealTimeUpdate as y,TEACHER_ACTOR_CLASS as O,OBSERVER_ACTOR_CLASS as h,Button as E,FpsCounter as T,KeyboardControlList as _,serializePlayerAction as r,TEACHER_NOOP_ACTION as g,Space as v,SimplePlay as w}from"@cogment/cogment-verse";import{jsx as s,jsxs as f}from"react/jsx-runtime";var o=new v({discrete:{n:2}}),L=({sendAction:e,fps:t=30,actorParams:l,...p})=>{let u=l?.className,[n,c]=A(!0),i=m(()=>c(C=>!C),[c]);N("p",i);let a=R(),P=m(C=>{if(a.size===0&&u===O){e(g);return}if(a.has("ArrowLeft")){e(r(o,0));return}else if(a.has("ArrowRight")){e(r(o,1));return}e(r(o,0))},[a,e,u]),{currentFps:d}=y(P,t,n);return f("div",{...p,children:[f("div",{className:"flex flex-row gap-1",children:[s(E,{className:"flex-1",onClick:i,children:n?"Resume":"Pause"}),s(T,{className:"flex-none w-fit",value:d})]}),s(_,{items:[["Left/Right Arrows","push cart to the left/right"],["p","Pause/Unpause"]]})]})},x=({actorParams:e,...t})=>e?.className===h?s(S,{actorParams:e,...t}):s(w,{actorParams:e,...t,controls:L}),k=x;export{k as default};
```

Comment:
For gym web components, why do we have all files as Gym[...] and without Gym?

Reply:
Hum that's because I forgot to clean it up :)

Reply:
fixed